.github/workflows/stale-pr.yml

```diff
@@ -0,0 +1,42 @@
+name: Stale Pull Requests
+
+on:
+  schedule:
+    - cron: '0 12 * * FRI' # Run every Friday at 12:00 (UTC)
```

Comment:
We can run it every day I believe.

Reply:
I have chosen this configuration to reduce the amount of jobs for CI

Reply:
Yes, I think we should run daily because +7 would make an issue double stale =D

Also, since this will remove a stale label if there is a comment/activity, we should remove it quickly:

```
# Remove `stale` label from pull requests on updates/comments
        remove-pr-stale-when-updated: true
```